app/api/services/informationextraction/specs/InformationExtraction.spec.ts

```diff
@@ -121,13 +121,13 @@ describe('InformationExtraction', () => {
             width: 457,
             height: 15,
             page_number: 1,
-            text: 'something something',
+            text: 'something',
```

Comment:
Probably another test case with a date type would be nice, because of the already mentioned type reasons.